cdm/src/main/java/ucar/nc2/dataset/conv/CF1Convention.java

```diff
@@ -177,10 +177,8 @@ public void augmentDataset(NetcdfDataset ds, CancelTask cancelTask) throws IOExc
       	v.addAttribute(new Attribute(CF.NODE_COUNT, ds.findAttValueIgnoreCase(v, CF.NODE_COUNT, "")));
       	v.addAttribute(new Attribute(CF.NODE_COORDINATES, ds.findAttValueIgnoreCase(v, CF.NODE_COORDINATES, "")));
       	v.addAttribute(new Attribute(CF.PART_NODE_COUNT, ds.findAttValueIgnoreCase(v, CF.PART_NODE_COUNT, "")));
-//      	if (ds.findAttValueIgnoreCase(v, CF.GEOMETRY_TYPE, "") != null) {
-      		if (ds.findAttValueIgnoreCase(v, CF.GEOMETRY_TYPE, "").equalsIgnoreCase("polygon")){
-      			v.addAttribute(new Attribute(CF.INTERIOR_RING, ds.findAttValueIgnoreCase(v, CF.INTERIOR_RING, "")));
-//      		}
+      	if (ds.findAttValueIgnoreCase(v, CF.GEOMETRY_TYPE, "").equalsIgnoreCase("polygon")){

```

Comment:
Is null-checking relevant at this point, or are we guaranteed to have that attribute if we reach this point?

Is it relevant to left-hand the literal for purposes of null safety? Example:
```
"polygon".equalsIgnoreCase(ds.findAttValueIgnoreCase(v, CF.GEOMETRY_TYPE, ""))
```

Related:
https://www.carlschroedl.com/blog/comp/left-handing-literals/2011/06/22/